libs/designsystem/src/lib/components/dropdown/dropdown.component.spec.ts

```diff
@@ -253,6 +258,33 @@ describe('DropdownComponent', () => {
       });
     });
 
+    describe('when configured with cardAligment', () => {
+      it('should render card with left alignment', () => {
+        spectator.component.cardAlign = cardAlignment.left;
+        spectator.component['state'] = openState.open;
+        spectator.detectChanges();
+        const buttonRect = buttonElement.getBoundingClientRect();
+        const card = spectator.query('kirby-card');
+        const cardRect = card.getBoundingClientRect();
+        expect(cardRect.left).toEqual(buttonRect.left);
```

Comment:
Consider adding linebreaks in a arrange-act-assert pattern:
```suggestion
        spectator.component.cardAlign = cardAlignment.left;
        spectator.component['state'] = openState.open;
        spectator.detectChanges();
        
        const buttonRect = buttonElement.getBoundingClientRect();
        const card = spectator.query('kirby-card');
        const cardRect = card.getBoundingClientRect();
        expect(cardRect.left).toEqual(buttonRect.left);
```

Reply:
Done